js/humanColors.js

```diff
@@ -0,0 +1,143 @@
+Colors = {
```

Comment:
For reasonable colours for humans with good contrast between them, I realised this is better than picking a random colour.